app/src/main/java/com/example/hodhod/hotels/network/HotelApi.java

```diff
@@ -0,0 +1,15 @@
+package com.example.hodhod.hotels.network;
+
+import com.example.hodhod.hotels.models.Hotel;
+
+import java.util.Observable;
+
+import io.reactivex.Flowable;
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+
+public interface HotelApi {
+
+    @GET("hotels")
+    Flowable<Hotel> getHotel();
```

Comment:
use Single instead of Flowable when we have request/response pattern
we use Flowable with back pressure which mean that observable omit data faster than subscriber ability